app/containers/TopBarContainer.js

```diff
@@ -15,24 +19,30 @@ type Props = InjectedProps;
 export default class TopBarContainer extends Component<Props> {
 
   render() {
-    const { actions, stores } = this.props;
-    const { app, topbar, theme } = stores;
+    try {
```

Comment:
this seems off, because I couldn’t find any element inside the try that throws

Reply:
@nicarq 
Sometimes this was being called before `stores` get initialized, but we changed Theme loading logic. Have not seen this problem in new code.